corehq/apps/es/migrations/0013_add_formLoadTime_to_form_meta.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 4.2.11 on 2024-06-25 14:36
+
+import corehq.apps.es.migration_operations
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('es', '0012_add_new_index_for_bha'),
+    ]
+
+    operations = [
+        corehq.apps.es.migration_operations.UpdateIndexMapping(
```

Comment:
Ahh! Another thing, since we have created the new indices that we would using for ES 6 migration, you would need to add another `UpdateIndexMapping` block manually in `operations` array.

It would look something like

```

        corehq.apps.es.migration_operations.UpdateIndexMapping(
            name=index_runtime_name('forms-2024-05-09'),
            type_='xform',
            properties={
            ....
           }, 
           es_versions=[],
          )
```
`forms-2024-05-09` is the name of new index. You can find the info [here ](https://github.com/dimagi/commcare-hq/blob/acd1658fd7030bcfea1b14fd8890d40bd4be1a30/corehq/apps/es/const.py#L47-L49)